sw/tools/parrot/bebop/wifi_tools/scripts/pprz.conf

```diff
@@ -0,0 +1,8 @@
+# Config file for PPRZ init script
+START_PPRZ=0
```

Comment:
Where is this file used? Why a pprz.conf and a scripts/pprz.conf?

Reply:
Going over the scripts it actually seems that both pprz.conf and scripts/pprz.conf could be superfluous, with pprz_swarmhub.conf being sufficient. I'll double check soon and make a new pull request, either deleting them or making the distinction more clear.